src/lib/holocene/menu/menu-item.svelte

```diff
@@ -40,6 +41,7 @@
   export let href = null;
   export let description: string = null;
   export let centered = false;
+  export let hover = true;
```

Comment:
Not sure about calling this `hover`, so open to feedback. Just want some way of making the hover styling here optional.

Reply:
Maybe hoverable?

Reply:
Yeah that's slightly better, [updated](https://github.com/temporalio/ui/pull/2144/commits/34130a1a54c1e1cfdaad9ba1843f5ec155a28a88) ✔️